### PR TITLE
Add macro to initialize ping response timeout

### PR DESCRIPTION
**Pull request by muneebahmed10**
*Description of changes:*
The `pingRespTimeoutMs` field of MQTTContext_t is initialized to 0 in MQTT_Init(), which doesn't make sense. This adds a macro to give it default value.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: libraries/standard/mqtt/src/mqtt.c [hunk 1]
```diff
@@ -1403,6 +1411,9 @@ MQTTStatus_t MQTT_Init( MQTTContext_t * pContext,
         pContext->appCallback = userCallback;
         pContext->networkBuffer = *pNetworkBuffer;
 
+        /* Set ping response timeout for process loop. */
+        pContext->pingRespTimeoutMs = MQTT_DEFAULT_PINGRESP_TIMEOUT_MS;
```
Comment (aggarw13) [inline libraries/standard/mqtt/src/mqtt.c:1415]: If we are adding a config macro for the ping response timeout, I think that the timeout should apply across all MQTT connections. 
If the ping response timeout is specific to the connection, then it should be an input parameter to `MQTT_Init`

Comment (muneebahmed10) [reply libraries/standard/mqtt/src/mqtt.c]: Removed the field from the context and made it just a library wide macro

File: libraries/standard/mqtt/src/mqtt.c [hunk 2]
```diff
@@ -41,6 +41,14 @@
     #define MQTT_MAX_CONNACK_RECEIVE_RETRY_COUNT    ( 5U )
 #endif
 
+/**
+ * @brief Number of milliseconds to wait for a ping response.
```
Comment (aggarw13) [inline libraries/standard/mqtt/src/mqtt.c:45]: Would be goof to mention that it is related to keep-alive in the comment.
```suggestion
 * @brief Number of milliseconds to wait for a ping response packet when a ping request packet is sent.
```

Comment (muneebahmed10) [reply libraries/standard/mqtt/src/mqtt.c]: Changed

File: libraries/standard/mqtt/src/mqtt.c [hunk 3]
```diff
@@ -41,6 +41,18 @@
     #define MQTT_MAX_CONNACK_RECEIVE_RETRY_COUNT    ( 5U )
 #endif
 
+/**
```
Comment (aggarg) [inline libraries/standard/mqtt/src/mqtt.c:44]: The description for MQTT_PINGRESP_TIMEOUT_MS should be same at all the places?

Comment (muneebahmed10) [reply libraries/standard/mqtt/src/mqtt.c:44]: Changed all the configs to use the same text as this file

**Pull request merged**